packages/shorebird_cli/lib/src/commands/release_new/releaser.dart

```diff
@@ -0,0 +1,72 @@
+import 'dart:io';
+
+import 'package:args/args.dart';
+import 'package:shorebird_cli/src/commands/release_new/release_type.dart';
+import 'package:shorebird_cli/src/shorebird_env.dart';
+import 'package:shorebird_code_push_client/shorebird_code_push_client.dart';
+import 'package:shorebird_code_push_protocol/shorebird_code_push_protocol.dart';
+
+/// {@template releaser}
+/// Executes platform-specific functionality to create a release.
+/// {@endtemplate}
+abstract class Releaser {
+  /// {@macro releaser}
+  Releaser({
+    required this.argResults,
+    required this.flavor,
+    required this.target,
+  });
+
+  /// The arguments passed to the command.
+  final ArgResults argResults;
+
+  /// The flavor of the release, if any.
+  final String? flavor;
+
+  /// The target script to run, if any.
+  final String? target;
```

Comment:
It's not immediately clear to me what this is referring to?

Reply:
Updated the comment here and for flavor.